repos/fdbt-site/src/pages/viewPurchaseMethods.tsx

```diff
@@ -16,32 +20,147 @@ const description =
+                        <p className="govuk-body govuk-!-margin-bottom-8">
+                            Define the way your tickets are sold, including where they are bought, the payment method
+                            and format
+                        </p>
+
+                        <div>
+                            {!purchaseMethods.length ? (
+                                <NoPurchaseMethods title="Purchase methods" />
+                            ) : (
+                                <PurchaseMethods
+                                    title="Normal purchase methods"
+                                    purchaseMethods={purchaseMethods}
+                                    isCapped={false}
+                                    deleteActionHandler={deleteActionHandler}
+                                />
+                            )}
+
+                            <a className="govuk-button" data-module="govuk-button" href="/managePurchaseMethod">
+                                Add a purchase method
+                            </a>
+                        </div>
+
+                        <div className="govuk-!-margin-top-4">
+                            {!cappedPurchaseMethods.length ? (
+                                <NoPurchaseMethods title="Capped purchase methods" />
+                            ) : (
+                                <PurchaseMethods
+                                    title="Capped purchase methods"
+                                    purchaseMethods={cappedPurchaseMethods}
+                                    isCapped={true}
+                                    deleteActionHandler={deleteActionHandler}
+                                />
+                            )}
+
+                            <a
+                                className="govuk-button"
+                                data-module="govuk-button"
+                                href="/managePurchaseMethod?isCapped=true"
+                            >
+                                Add a capped purchase method
+                            </a>
+                        </div>
+
+                        {popUpState && (
+                            <DeleteConfirmationPopup
+                                entityName={popUpState.entityName}
+                                deleteUrl={buildDeleteUrl(popUpState.entityId, csrfToken)}
+                                cancelActionHandler={cancelActionHandler}
+                            />
+                        )}
+                    </div>
+                </div>
+            </BaseLayout>
         </>
     );
 };
 
+interface NoPurchaseMethodsProps {
+    title: string;
+}
+const NoPurchaseMethods = ({ title }: NoPurchaseMethodsProps): ReactElement => {
+    return (
+        <>
+            <h2 className="govuk-heading-l">{title}</h2>
+            <p className="govuk-body">
+                <em>You currently have no purchase methods saved.</em>
```

Comment:
We're  showing this even when its just for capped purchase methods. It needs to say 'You currently have no capped purchase methods saved.' when they have no capped purchase methods.